masonite/helpers/routes.py

```diff
@@ -34,3 +34,16 @@ def patch(url, controller):
     Shortcut for PATCH http
     """
     return Patch().route(url, controller)
+
+
+def group(url, route_list):
+    """
+    Group route
+    """
+    if url[-1:] != '/':
```

Comment:
what is this here? this looks really confusing. Are you checking if the url starts with a forward slash?

If so you should just do something like:

```python
if not url.startswith('/'):
    # do logic
```

but even still I don't think we should check for this. We don't check in any other routes. All routes should start with a slash anyway. If we start doing this check here then we need to perform this check on all routes. If we are going to make this change then I say we do it in some 2.x release instead.

Reply:
No, I only check if it has "/" at the end. I erase it?

Reply:
oh ok ummm then that's fine actually. But then do `url.endswith('/')`

Reply:
actually you are appending a `/` here. Don't do that. Just get rid of this actually

Reply:
Ok